web/templates/sapsystems.html.tmpl

```diff
@@ -1,10 +1,94 @@
+        </div>
+    </div>
+    <hr class="margin-10px"/>
+    <h5>Filters</h5>
+    <div class="horizontal-container">
+        <script>
+            $(document).ready(function () {
+                {{- range $Key, $Value := .AppliedFilters }}
+                $("#{{ $Key }}").selectpicker("val", {{ $Value }});
+                {{- end }}
+                $('#clean').click(function () {
+                    $('.selectpicker').selectpicker("deselectAll")
+                });
+            });
+        </script>
+        {{/* {{ template "health_filter" }} */}}
+        <select name="sid" id="sid" class="selectpicker" multiple
+                data-selected-text-format="count > 3" data-actions-box="true" data-live-search="true" title="SID">
+            {{- range .SAPSystemsTable.GetAllSIDs }}
+                <option value="{{ . }}">{{ . }}</option>
+            {{- end }}
+        </select>
+    </div>
+    <div class='table-responsive'>
+        <table class='table eos-table'>
+            <thead>
+            <tr>
+                <th scope='col'></th>
+                <th scope='col'></th>
+                <th scope='col'>SID</th>
+                <th scope='col'>Description</th>
+                <th scope='col'>Tags</th>
+            </tr>
+            </thead>
+            <tbody>
+            {{- range $index, $value := .SAPSystemsTable }}
+                <tr>
+                    <td class="row-status">{{ template "health_icon" "unknown" }}</td>
+                    <td class="row-status"><a class="eos-icons eos-18  collapse-toggle clickable collapsed text-dark"
+                                              data-toggle="collapse" data-target="#inner_{{ $index }}"></a></td>
+                    <td><a href="/sapsystems/{{ .SID }}?environment=default&landscape=default">{{ .SID }}</a></td>
+                    <td></td>
+                    <td></td>
+                <tr id="inner_{{ $index }}" class="collapse">
+                    <td colspan="2"></td>
+                    <td colspan="3">
+                        <table class='table eos-table'>
+                            <thead>
+                            <tr>
+                                <th scope='col'></th>
+                                <th scope='col'>SID</th>
+                                <th scope='col'>Features</th>
+                                <th scope='col'>Instance number</th>
+                                <th scope='col'>Cluster name</th>
+                                <th scope='col'>Host</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {{- range .InstancesTable }}
+                            <tr>
+                                <td class="row-status">{{ template "health_icon" "unknown" }}</td>
+                                <td>
+                                    <a href="/sapsystems/{{ .SID }}?environment=default&landscape=default">{{ .SID }}</a>
```

Comment:
We will need to be careful with this `SID` link (in the inner table)
This SID could be HANA installation SID, which is not a SAP system. 
We would need to see what to do. 
Maybe, we should remove the link from the internal table

Reply:
if we remove the link and the HANA SID is different from the outer row, we would not have a link to the HANA SID

Reply:
Yes, I know. What this page should redirect to SAP systems, and a HANA installation is not a full SAP system.
I don't know if we will have pages for individual SAP instances or databases honestly

Reply:
Ok I was missing this context, i will remove the link in the inner rows then